lit_gpt/lora.py

```diff
@@ -402,9 +386,7 @@ def T(w):
                 after_A.transpose(-2, -1),  # (64, 64, 4) -> (64, 4, 64)
                 self.lora_B.unsqueeze(-1),  # (256, 2) -> (256, 2, 1)
                 groups=sum(self.enable_lora),
-            ).transpose(
-                -2, -1
-            )  # (64, 4, 64) @ (256, 2, 1) -> (64, 256, 64) -> (64, 64, 256)
+            ).transpose(-2, -1)  # (64, 4, 64) @ (256, 2, 1) -> (64, 256, 64) -> (64, 64, 256)
```

Comment:
I also have a question regarding `.transpose(-2, -1)`. 
It has a shorter equivalent - [`.mT`](https://pytorch.org/docs/stable/tensors.html#torch.Tensor.mT), but since `.transpose` is used everywhere it looks like it's better to stick to it.
Any thought on it?

Reply:
My personal preference is to keep the transpose, spelling out what it does should help people who are onboarding and learning the operations in a transformer model.